.github/workflows/terraform-plan-apply.yml

```diff
@@ -27,25 +32,29 @@ jobs:
           set -o nounset
           # set -o xtrace
 
-          git fetch origin main:localmain
+          if [[ -z "${{github.event.inputs.trialDefinitionPath}}}" ]]; then
```

Comment:
can you add a comment such as:
"if a specific trial was requested, we deploy it. meaning this was an external trigger (not a pr). other wise we deploy only changed files"

Reply:
Added comment